index.es.md

```diff
@@ -0,0 +1,104 @@
+---
+# Translation info https://www.w3.org/wiki/WAI/Website/Translate
+
+title: "Visión general sobre la evaluación de la accesibilidad web"
+nav_title: "Visión general sobre la evaluación"
+
+lang: en
+last_updated: 2018-09-20
+permalink: /test-evaluate/es
+
+
+translators:
+- name: "Jorge Rumoroso"
+  link: "https://twitter.com/rumoroso"
+contributors:
+#- name: ""
+
+github: 
+  repository: w3c/wai-eval-overview
+  path: index.es.md
+
+feedbackmail: wai@w3.org
+footer: >
+  <p><strong>Fecha:</strong> Actualizado el 20 de Septiembre de 2018.</p>
+  <p><strong>Editora:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Desarrollado con la colaboración del Grupo de Trabajo de Educación y Divulgación (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+ref: /test-evaluate/
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Resumen" class="full" %}
+{:/}
+
+Esta página contiene enlaces a recursos para ayudar a evaluar la accesibilidad web. La evaluación de la accesibilidad también se denomina "análisis", "auditoría" o "prueba".
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenidos de la página" class="full" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Introducción {#intro}
+
+Al desarrollar o rediseñar un sitio web o una aplicación web, evalúe la accesibilidad desde el principio y durante todo el proceso de desarrollo para identificar los problemas de accesibilidad desde el principio, cuando es más fácil resolverlos.
```

Comment:
In order to avoid the repetition of  "desde el principio", I'd replace one of them with "lo antes posible", for example.